docs/guides/getting-started-java-spring.md

```diff
@@ -182,6 +186,7 @@ In your terminal, run `mvn spring-boot:run`, where you should see the `charging
 To start a process instance programmatically, take the following steps:
 
 1. In `ProcessPaymentsApplication.java`, convert the application to a `CommandLineRunner`, by adding `implements CommandLineRunner` to the `ProcessPaymentsApplication` class declaration. Instantiate a static `Logger` variable, and an instance variable named `zeebeClient` with the `@Autowired` annotation.
+   1. Additionally, decorate the `ProcessPaymentsApplication` class with `@Deployment(resources = "classpath:process-payments.bpmn")` in `ProcessPaymentsApplication.java`:
```

Comment:
blocking: The main concern with the `@Deployment` is that it shouldn't be added in this step -- it should be added in the next step, "Deploy the process," because that's when it's used. We should remove this line, and also remove the `@Deployment` line in this step's code sample. Then, in step 8, revert the removal of the `@Deployment` line.